src/rpc/blockchain.cpp

```diff
@@ -1155,6 +1156,7 @@ UniValue getblockchaininfo(const UniValue& params, bool fHelp)
     valuePools.push_back(ValuePoolDesc("sprout", tip->nChainSproutValue, std::nullopt));
     valuePools.push_back(ValuePoolDesc("sapling", tip->nChainSaplingValue, std::nullopt));
     valuePools.push_back(ValuePoolDesc("orchard", tip->nChainOrchardValue, std::nullopt));
+    valuePools.push_back(ValuePoolDesc("lockbox", tip->nChainLockboxValue, std::nullopt));
```

Comment:
Not sure we want this to be named `lockbox` in the RPC output indefinitely, but I guess it doesn't matter too much.

Reply:
I agree; I made this change to be consistent with what ZF requested.